newm-server/src/main/kotlin/io/newm/server/features/distribution/eveara/EvearaDistributionRepositoryImpl.kt

```diff
@@ -998,7 +998,7 @@ class EvearaDistributionRepositoryImpl(
                         productFormat = "stereo",
                         coverImage =
                             CoverImage(
-                                url = release.coverArtUrl.asValidUrl(),
+                                url = release.coverArtUrl.asValidUrl().replace("c_limit,w_4000,h_4000/", ""),
```

Comment:
Are these params always in that order? If not, we may need to do something more elaborated like parsing the URL, and re-generate it without them.

Reply:
These are hard-coded in the newm-web code, so I believe this is correct.